ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_rounding.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from hypothesis import given, strategies as st
+
+import ivy
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy.current_backend().valid_float_dtypes, min_num_dims=1
```

Comment:
instead of `ivy.current_backend().valid_float_dtypes` can you please change as done [here](https://github.com/unifyai/ivy/blob/30431311bb634141d55b90a684d716d6eb2578e4/ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_logical_operations.py#L16)

Reply:
Is this a new function bcs it says?

E   AttributeError: module 'ivy_tests.test_ivy.helpers' has no attribute 'get_dtypes'

Reply:
After I updated my branch, now "helpers.get_dtypes("float", full=True)"  is giving this error. I am not sure what it means or what's causes.

Reply:
After I updated by branch, I have this error with get_dtypes(kind='float', full=True)

E hypothesis.errors.InvalidArgument: Cannot sample from get_dtypes(kind='float', full=True), not an ordered collection. Hypothesis goes to some length to ensure that the sampled_from strategy has stable results between runs. To replay a saved example, the sampled values must have the same iteration order on every run - ruling out sets, dicts, etc due to hash randomization. Most cases can simply use `sorted(values)`, but mixed types or special values such as math.nan require careful handling - and note that when simplifying an example, Hypothesis treats earlier values as simpler.

Reply:
that's weird because it has been used in many functions

Reply:
that's weird because it's been used in multiple functions